image-trigger

```diff
@@ -44,7 +44,6 @@ REFRESH = {
     "debian-stable": {},
     "fedora-41": {},
     "fedora-42": {},
-    "fedora-42-boot": {},
```

Comment:
FTR, I didn't forget to drop 42-anaconda-payload here. It was never here in the first place.